src/providers/ipa/ipa_subdomains.c

```diff
@@ -2762,6 +2762,7 @@ static void ipa_subdomains_refresh_certmap_done(struct tevent_req *subreq)
         /* Not good, but let's try to continue with other server side options */
     }
 
+#ifdef BUILD_PASSKEY
```

Comment:
`#include "providers/ipa/ipa_subdomains_passkey.h"`
at the beginning of this file also can be wrapped in `#ifdef BUILD_PASSKEY`

Reply:
Fixed.

Reply:
Thanks.